spec/support/authorization_stubs.rb

```diff
@@ -3,7 +3,7 @@ module AuthorizationStubs
 
   def allow_operation(operation, *args, authorizer: instance_double(AUTHORIZER_CLASS), **kwargs)
     allow(authorizer).to receive(operation).tap {|x|
-      if kwargs.empty? then x.with(*args) else x.with(*args, **kwargs) end
+      kwargs.empty? ? x.with(*args) : x.with(*args, **kwargs)
```

Comment:
I am a bit sad about this change, but I see the benefit of having the style rules enforced automatically & not spending time agonising over lots of custom rules (as I probably now will while adding this setup to my own project).

Reply:
Yeah, code styles can differ between projects. I'm myself happy about this change, as I prefer ternaries over one line `if ... then ... else ... end` statements 😛